src/admin/systemTypes/systemTypes.component.tsx

```diff
@@ -130,21 +166,64 @@ function SystemTypes() {
     // Functions
     ...onPreservedStatesChange,
 
-    renderTopToolbarCustomActions: ({ table }) => (
-      <Box>
-        <Button
-          startIcon={<ClearIcon />}
-          sx={{ mx: '4px' }}
-          variant="outlined"
-          disabled={preservedState.columnFilters.length === 0}
-          onClick={() => {
-            table.resetColumnFilters();
-          }}
-        >
-          Clear Filters
-        </Button>
-      </Box>
-    ),
+    renderTopToolbarCustomActions: ({ table }) => {
+      function isExactFilterActive(
+        expectedFilters: { id: string; filterFn?: string; value: string }[]
+      ) {
+        const actualFilters = table.getState().columnFilters;
+        const actualFilterFns = table.getState().columnFilterFns;
+
+        // Check length matches
+        if (actualFilters.length !== expectedFilters.length) return false;
+
+        // Check every expected filter matches actual filter and filterFn
+        return expectedFilters.every(({ id, filterFn, value }) => {
+          const actualFilter = actualFilters.find((f) => f.id === id);
+          if (!actualFilter) return false;
+          if (actualFilterFns[id] !== filterFn) return false;
+          // Compare values stringified (arrays)
+
+          return JSON.stringify(actualFilter.value) === JSON.stringify(value);
+        });
+      }
+      return (
+        <Box>
+          <Button
+            startIcon={<ClearIcon />}
+            sx={{ mx: '4px' }}
+            variant="outlined"
+            disabled={preservedState.columnFilters.length === 0}
+            onClick={() => {
+              table.resetColumnFilters();
+            }}
+          >
+            Clear Filters
+          </Button>
+
+          <Button
+            sx={{ mx: 0.5 }}
+            variant="outlined"
+            disabled={isExactFilterActive([
+              {
+                id: 'isSpare',
+                value: COLUMN_FILTER_BOOLEAN_OPTIONS[0],
+              },
+            ])}
+            onClick={() => {
+              table.resetGlobalFilter();
+              table.setColumnFilters([
+                {
+                  id: 'isSpare',
+                  value: COLUMN_FILTER_BOOLEAN_OPTIONS[0],
+                },
+              ]);
+            }}
+          >
+            Spares Definition
```

Comment:
```suggestion
            View spares definition
```
I don't think its clear what the button is for without saying view?

Reply:
I will use `Show` instead of `View` as this is used in the Rules table. 
<img width="1912" height="803" alt="image" src="https://github.com/user-attachments/assets/7eb689d2-41ed-4a77-952b-38fc3ca1c2f2" />